sdk/plugin/grpc_system_test.go

```diff
@@ -183,6 +183,34 @@ func TestSystem_GRPC_entityInfo(t *testing.T) {
 	}
 }
 
+func TestSystem_GRPC_groupsForEntity(t *testing.T) {
+	sys := logical.TestSystemView()
+	sys.GroupsVal = []*logical.Group{
+		{
+			ID:   "group1-id",
+			Name: "group1",
+			Metadata: map[string]string{
+				"group-metadata": "metadata-value",
+			},
+		},
+	}
+	client, _ := plugin.TestGRPCConn(t, func(s *grpc.Server) {
+		pb.RegisterSystemViewServer(s, &gRPCSystemViewServer{
+			impl: sys,
+		})
+	})
+	defer client.Close()
+	testSystemView := newGRPCSystemView(client)
+
+	actual, err := testSystemView.GroupsForEntity("")
```

Comment:
I'm confused by the empty string argument here, could you explain please?  Shouldn't it be `group1-id`?

Reply:
Oh nvm, I see the TestSystemView implementation ignores its arg.